### PR TITLE
add static svg sprite sheet file

### PR DESCRIPTION
**Pull request by dstaley**
### :pushpin: Summary

This PR adds support for a plain SVG sprite sheet (in addition to the existing JavaScript module version).

### :hammer_and_wrench: Detailed description

In order to support using the SVG sprite sheet in our Next.js websites without shipping the sprite sheet as several hundred kilobytes of JavaScript, we'd like to simply reference an SVG file. This PR outputs the generated SVG sprite sheet to an SVG file in addition to the existing JS module. Two attributes were added to the root `<svg>` element for maximum compatibility:
1. `xmlns="http://www.w3.org/2000/svg"` Some browsers will not correctly render remote `<use>` references unless the remote document has the correct namespace.
2. `viewBox="0 0 0 0"` Next.js expects there to be a `viewBox` attribute to calculate dimensions. While this isn't actually useful in this case, it does prevent Next.js from throwing an error about being unable to parse the file.

Neither of these changes should have any impact on existing usage, hence the `minor` version bump.
 
### :link: External links

Jira ticket: [HDS-XXX](https://hashicorp.atlassian.net/browse/HDS-XXX)
Figma file: [if it applies]

***

### 👀 Component checklist

- [ ] Percy was checked for any visual regression
- [ ] A11y tests have been run locally (`yarn test:a11y --filter="COMPONENT-NAME"`)
- [ ] If documenting a new component, an acceptance test that includes the `a11yAudit` has been added
- [x] A changelog entry was added via [Changesets](https://github.com/changesets/changesets) if needed (see [templates here](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#templates-for-npm-packages))

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: packages/flight-icons/scripts/build-parts/generateBundleSVGSprite.ts [hunk 1]
```diff
@@ -26,7 +26,14 @@ export async function generateBundleSVGSprite({ config, catalog } : { config: Co
     const sprites = svgstore({
         // see https://github.com/svgstore/svgstore#options for details
         renameDefs: false, // we already create unique IDs (using the icon name) in the SVGO step
-        svgAttrs: { width: '0', height: '0', class: 'flight-sprite-container', 'aria-hidden': 'true'  },
+        svgAttrs: { 
+            width: '0',
+            height: '0',
+            xmlns: "http://www.w3.org/2000/svg",
```
Comment (didoo) [inline packages/flight-icons/scripts/build-parts/generateBundleSVGSprite.ts:32]:
```suggestion
            // some browsers will not correctly render remote <use> references unless the remote document has the correct namespace
            xmlns: "http://www.w3.org/2000/svg",
```

File: packages/flight-icons/scripts/build-parts/generateBundleSVGSprite.ts [hunk 2]
```diff
@@ -26,7 +26,14 @@ export async function generateBundleSVGSprite({ config, catalog } : { config: Co
...
+            viewBox: "0 0 0 0",
```
Comment (didoo) [inline packages/flight-icons/scripts/build-parts/generateBundleSVGSprite.ts:33]:
```suggestion
            // Next.js expects there to be a viewBox attribute to calculate dimensions; while this isn't actually useful in the case of an SVG sprite, it does prevent Next.js from throwing an error about being unable to parse the file
            viewBox: "0 0 0 0",
```

**Review by didoo — APPROVED** [review]
@dstaley I've added a couple of suggestions (adding comments to the code for future reference) but apart from that... all good to go! 👍

Thanks for the contribution! 🙏

File: .changeset/mighty-singers-prove.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Add static SVG sprite sheet file
```
Comment (alex-ju) [inline .changeset/mighty-singers-prove.md:5]: nitpick: we've just updated [our changelog guidance](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#general-guidance) to use past tense in changelog entries
```suggestion
Added static SVG sprite sheet file
```

**Review by alex-ju — APPROVED** [review]
The `xmlns` and `viewBox` changes look good! Assuming you tested the `svg-sprite.svg` in your project I'm happy to get it in.

**Review by MelSumner — APPROVED** [review]
This looks good to me, thank you for proposing something that works best for your use case.

**Pull request merged**